core/hydromet_stratified.py

```diff
@@ -0,0 +1,560 @@
+import pathlib as pl
+import numpy as np
+import pandas as pd
+from scipy.integrate import quad
+from scipy.optimize import minimize
+from scipy import interpolate, stats, special
+from matplotlib import pyplot as plt
+from cycler import cycler
+
+
+#----------------------------------------------------------------------------------------------------------------------#
+# Functions called by EventTable_Stratified.ipynb.
+#----------------------------------------------------------------------------------------------------------------------#
+
+def Q_SCS(R: np.array, CN: float, mu: float) -> float:
```

Comment:
Let's make the R variable more meaningful, e.g. 'rainfall_total' and then add a comment with the units and any additional info. The description could be then copied to better comment the other function docstrings.

Reply:
Added to issue #47.